src/main/java/uk/nhs/digital/nhsconnect/lab/results/translator/mapper/BundleMapper.java

```diff
@@ -51,15 +65,16 @@ public Bundle mapToBundle(final PathologyRecord pathologyRecord) {
     }
 
     private Bundle generateInitialPathologyBundle() {
-        Bundle bundle = new Bundle();
+        final Bundle bundle = new Bundle();
 
+        bundle.setId(uuidGenerator.generateUUID());
         bundle.setMeta(new Meta()
-                .setLastUpdatedElement((InstantType.now()))
-                .setProfile(BUNDLE_META_PROFILE)
+            .setLastUpdatedElement((InstantType.now()))
+            .setProfile(BUNDLE_META_PROFILE)
```

Comment:
We need to make sure everyone has the same formatting settings to avoid inconsequential changes like these flooding the PRs.